haskell-overlays/untriaged.nix

```diff
@@ -1,22 +1,73 @@
 { haskellLib
 , fetchFromGitHub
 , enableLibraryProfiling
+, nixpkgs
 }:
-
 with haskellLib;
+let addGIDeps = p: extraBuildInputs: girSearchPathPackages: p.overrideAttrs (drv: {
```

Comment:
Can we open (or maybe reopen) an issue in cabal2nix for this?

Reply:
Added to the todo list